serverless/tensorflow/faster_rcnn_inception_v2_coco/nuclio/main.py

```diff
@@ -0,0 +1,48 @@
+import json
+import base64
+import io
+from PIL import Image
+import yaml
+from model_loader import ModelLoader
+
+
+def init_context(context):
+    context.logger.info("Init context...  0%")
+    model_path = "/opt/nuclio/faster_rcnn/frozen_inference_graph.pb"
+    model_handler = ModelLoader(model_path)
```

Comment:
Codacy found an issue: [No value for argument 'weights' in constructor call](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5729179)